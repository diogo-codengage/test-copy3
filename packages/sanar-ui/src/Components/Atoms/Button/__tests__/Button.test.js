import React from "react";
import ReactDOM from "react-dom";
import ESButton from "../Button.jsx";

it("renders correctly", () => {
	const component = <ESButton>Button</ESButton>;

	const tree = TestRenderer.create(component).toJSON();
	expect(tree).toMatchSnapshot();
});
