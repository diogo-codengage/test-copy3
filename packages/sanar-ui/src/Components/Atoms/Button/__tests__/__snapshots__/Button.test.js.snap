// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<button
  className="ant-btn es-button"
  onClick={[Function]}
  type="button"
>
  <span>
    Button
  </span>
</button>
`;
